meinberlin/assets/scss/components/_list.scss

```diff
@@ -0,0 +1,10 @@
+.list--highlight {
+    padding-bottom: 2em;
+
+    background-color: $bg-tertiary;
+
+    .filter-bar:first-child {
+        position: relative;
+        top: -0.8em;
+    }
```

Comment:
Can we convert this to a filter bar modifier?

Also: How does this work with several lines?